lib/src/tagging/hash.spec.ts

```diff
@@ -0,0 +1,20 @@
+import { cleanse } from "../cleansers/cleanser";
+import { parse } from "../parsers/parser";
+import { hashContractDefinition } from "./hash";
+
+describe("Hash", () => {
+  describe("hashContractDefinition", () => {
+    it("returns a new hash when the contract is updated", () => {
```

Comment:
I think we need more extensive test coverage here.

For example, you could implement ` hashContractDefinition` as follows:
```
function hashContractDefinition(contract) {
  return hash(contract.api.name);
}
```

This test would pass, but the underlying hash wouldn't actually be good enough.

I'd recommend adding tests for:
- adding a new endpoint
- changing a property in request body
- changing a property in response body
- possibly also header and request parameters

Reply:
Thank you for the feedback. Will review this.